charts/deepgram-self-hosted/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 
 ## [Unreleased]
 
+### Changed
+
+- AWS samples updated to take advantage of new [EKS accelerated AMIs](https://github.com/awslabs/amazon-eks-ami/releases/tag/v20240928), which bundle the required NVIDIA driver and toolkit instead of being installed by the NVIDIA GPU operator 
```

Comment:
I found this exact link not super enlightening - maybe link to the [top-level repo](https://github.com/awslabs/amazon-eks-ami), and/or their [changelog post](https://aws.amazon.com/about-aws/whats-new/2024/10/amazon-eks-nvidia-aws-neuron-instance-types-al2023/)?

Reply:
That changelog post is *exactly* what I wanted, but my Google searches weren't finding that post so I fell back to the GH page.

Reply:
3c0d68ad44b3